libs/commands/src/services/utilities/getCommandFromHandlerMetadata.ts

```diff
@@ -0,0 +1,21 @@
+import { Type } from '@nestjs/common';
+import { COMMAND_HANDLER_METADATA } from '../../decorators/constants';
+import { CommandHandlerMetadata } from '../../decorators/types/CommandHandlerMetadata';
+import { MetadataNotFoundException } from '../../exceptions/metadata-not-found-exception';
+import { ICommandHandler } from '../../interfaces/command-handler.interface';
+import { ICommand } from '../../interfaces/command.interface';
+
```

Comment:
You might want to unit test this.

Reply:
It's integration tested at a higher level. Add a unit test if it fails unexpectedly in the future. It's fairly simple logic.